js/src/views/Contract/contract.js

```diff
@@ -79,9 +83,17 @@ class Contract extends Component {
     }
   }
 
+  componentWillUnmount () {
+    const { api } = this.context;
+    const { subscriptionId, blockSubscriptionId, contract } = this.state;
+
+    api.unsubscribe('eth_blockNumber', blockSubscriptionId);
+    contract.unsubscribe(subscriptionId);
```

Comment:
Worried that what you did above was adding the eventName and here you are not using the eventName :)